templates/prelogin/_sections/home/build.html

```diff
@@ -90,7 +90,7 @@ <h2 class="text-center">
                     {% endblocktrans %}
                 </h3>
                 <p>
-                    {% url 'public_pricing' as pricing_url %}
+                    {% url 'public_pricing' LANGUAGE_CODE as pricing_url %}
```

Comment:
See https://github.com/dimagi/commcarehq-prelogin/pull/49/files#r48537563

Django can *reverse translate* the urls without us adding the `LANGUAGE_CODE` ourselves, but it has above mentioned limitations